src/js/track.js

```diff
@@ -0,0 +1,83 @@
+var _ = require('underscore');
+var d3 = require('d3');
+var $ = require('jquery');
+var D3SVGCellRenderer = require('./d3_svg_cell_renderer');
+var utils = require('./utils');
+
+module.exports = {};
+
+var defaultTrackConfig = {
+	label: 'Gene',
+	id_member: 'sample',
+	cell_height: 20,
+	track_height: 20,
+	track_padding: 2.5,
+}; 
+
+function Track(name, oncoprint, data, config) {
+	var self = this;
+	self.name = name;
+	self.config = $.extend({}, defaultTrackConfig, config || {}); // inherit from default
```

Comment:
opportunity to write a function instead of documentation: `function inherit_config_from_default(default, config)`.